enforcer-rules/src/site/apt/banDynamicVersions.apt.vm

```diff
@@ -0,0 +1,108 @@
+~~ Licensed to the Apache Software Foundation (ASF) under one
+~~ or more contributor license agreements.  See the NOTICE file
+~~ distributed with this work for additional information
+~~ regarding copyright ownership.  The ASF licenses this file
+~~ to you under the Apache License, Version 2.0 (the
+~~ "License"); you may not use this file except in compliance
+~~ with the License.  You may obtain a copy of the License at
+~~
+~~ http://www.apache.org/licenses/LICENSE-2.0
+~~
+~~ Unless required by applicable law or agreed to in writing,
+~~ software distributed under the License is distributed on an
+~~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+~~ KIND, either express or implied.  See the License for the
+~~ specific language governing permissions and limitations
+~~ under the License.    
+ 
+  ------
+  Ban Dynamic Versions
+  ------
+  Konrad Windszus
+  ------
+  October 2022
+  ------
+
+Ban Dynamic Versions
+
+  This rule bans dependencies having versions which require resolution (i.e. dynamic versions which might change with each build). Dynamic versions are either 
+  
+  * version ranges,
+  
+  * the special placeholders <<<LATEST>>>/<<<RELEASE>>> or
+  
+  * versions ending with <<<-SNAPSHOT>>>.
+
+  []
+
+  The following parameters are supported by this rule:
+  
+  * <<allowSnapshots>> - if <<<true>>> dependencies with versions ending with <<<-SNAPSHOT>>> will not be banned.  Default is <<<false>>>.
+  
+  * <<allowRelease>> - if <<<true>>> dependencies with version placeholder <<<RELEASE>>> will not be banned.  Default is <<<false>>>.
+  
+  * <<allowLatest>> - if <<<true>>> dependencies with versions placeholder <<<LATEST>>> will not be banned.  Default is <<<false>>>.
+  
+  * <<allowRanges>> - if <<<true>>> versions ending with <<<-SNAPSHOT>>> will not be banned.  Default is <<<false>>>.
+  
+  * <<checkOptionals>> - if <<<true>>> also optional dependencies will be checked.  Default is <<<false>>>.
+  
+  * <<excludedScopes>> - the list of scopes to exclude;
+  
+  * <<ignores>> - a list of dependencies to ignore. The format is <<<groupId[:artifactId][:version][:type][:scope][:classifier]>>> where <<<artifactId>>>, <<<version>>>, <<<type>>>, <<<scope>>> and <<<classifier>>> are optional. Wildcards may be used to replace an entire or just parts of a section.
```

Comment:
This does not correspond to https://github.com/apache/maven-enforcer/pull/187/files#diff-9dd3c382c00a8c5fa752798d8f5e7bae9b72fe5d7454bebf08dc77d1244fe5b4R113.

Reply:
This is actually the supported format, fixed the javadoc in https://github.com/apache/maven-enforcer/pull/187/commits/182722b981b2257ec56bd8b34c3d21e813380cd5.